app/routes/_gcn.circulars/circulars.server.ts

```diff
@@ -261,3 +291,198 @@ export async function circularRedirect(query: string) {
     throw redirect(circularURL)
   }
 }
+
+/**
+ * Gets a specific version of a given circular
+ *
+ * Throws an HTTP error if:
+ *  - The provided circularId isNaN
+ *  - The provided version isNaN
+ *  - No version is found matching the provided circularId and version
+ *
+ * @param circularId
+ * @param version
+ * @returns a Circular corresponding to the specific version and Id
+ */
+export async function getSpecificCircularVersion(
+  circularId: number,
+  version: number
+): Promise<Circular> {
+  if (isNaN(circularId) || isNaN(version))
+    throw new Response(null, { status: 404 })
+  const db = await tables()
+  const result = await db.circulars_history.get({
+    circularId,
+    version,
+  })
+  if (!result)
+    throw new Response(null, {
+      status: 404,
+    })
+  return result
+}
```

Comment:
Let's merge this functionality into the existing `get()` function, adding an optional version positional argument.

Also: since the _current_ version is not stored in the history table, if the caller requests the current version by its number, then we need to correctly handle that case.